### PR TITLE
TST: Windows and macOS

### LINKED ISSUE #3: Add support for Windows
Support for Windows is broken until https://github.com/audeering/audresample/pull/5 is merged.

### PR DESCRIPTION
**Pull request by frankenjoe**
Closes #3, #4

After fixing https://github.com/audeering/audresample/pull/5 and https://github.com/audeering/audresample/pull/10, we can now add tests for Windows and macOS here as well.

### CODE REVIEW

**Comment by hagenw** [comment]
OK, looks like the usual path errors under Windows

**Comment by frankenjoe** [comment]
> Could you add to the description, that it closes #3 and #4.

done

**Comment by frankenjoe** [comment]
> OK, looks like the usual path errors under Windows

Of course :)

**Comment by hagenw** [comment]
The diffs for the tests simply say you changed everything.

Could you maybe briefly summarized how you updated the tests?

**Comment by frankenjoe** [comment]
> The diffs for the tests simply say you changed everything.

I thought it would be smart to fix the tests on my VM to immediately see if it is working. But of course stupid Windows changed the line ending as well. Changed to LF again.

File: tests/test_process.py [hunk 1]
```diff
@@ -504,34 +504,17 @@ def test_process_folder(
     )
     sampling_rate = 8000
     path = str(tmpdir.mkdir('wav'))
-    files = [f'{path}/file{n}.wav' for n in range(num_files)]
-    rel_path = os.path.relpath(path)
```
Comment (frankenjoe) [inline tests/test_process.py:508]: For some reasons `os.path.relpath()` was not working in the Windows job (on my VM it did...). But I anyway find this test a bit messy and hard to read. I know simply compare if `process_folder()` gives the same result as `process_files()` on all files in the folder. Which is essentially what we want.

Comment (hagenw) [reply tests/test_process.py:508]: Yes, you are right, and it simplifies the test code a lot.

**Pull request merged**